### ISSUE TITLE
It is possible to see through walls

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1169]
When you have a fortified dungeon, you can see the hero parties trying to dig into your dungeon. See the screenshot below from the 20160402 build:

![seethrough](https://cloud.githubusercontent.com/assets/13840686/14764231/d1f44f3e-09b0-11e6-96da-85dfdb35b0ca.jpg)

**Comment by hwoarangmy** [comment]
Here, that's a feature, not a bug. IMHO, that allows to see enemy units trying to dig to your dungeon and I think it is pretty cool ^^

**Comment by Loobinex** [comment]
It's not cool at all, it's a glitch. You should be alerted some other way, like hearing a sound of pounding on walls or digging through dirt. This rewards the player who pays attention, and he can use the 'sight' spell to see the units there.

**Comment by Bertram25** [comment]
@Loobinex Hmm, please understand that here, it would make a common simple game behavior more complex just for this specific need.
I do myself think it is cool to be able to sense what is around your dungeon, and that fear grips on you when you see the dwarves dig around your dungeon.

What you want is a specific DK1 behaviour but it is not a glitch, IMHO. Is there any other interesting reason you may have in mind?

**Comment by Loobinex** [comment]
OK, if you actually intended this behavior, it is obviously not a glitch. I find it very silly to be able to see through walls, and would find not knowing what exactly is coming for me be both more logical and more ominous.

There's no deeper reason behind this, if you want to be able to see through your walls and be able to know what your enemy has without having to do anything for it, then no change is required and you can reject the issue.

**Comment by Bertram25** [comment]
> if you want to be able to see through your walls and be able to know what your enemy has without having to do anything for it, then no change is required and you can reject the issue.

I'll wait for others point of view, but IMHO, sensing 1 tile next to your walls is something I can understand. So I'd vote to keep it as it is.

**Comment by akien-mga** [comment]
Yeah, that's a behaviour you find in many other strategy games btw, like Age of Empires and the like. You often get some visibility beyond your walls so that you they can serve as a real strategic defense and not just an obstacle.

One could easily imagine that every piece of claimed wall is built with an arrow slit which lets creatures inside the dungeon see if there are enemies just behind the wall.

**Comment by hwoarangmy** [comment]
> One could easily imagine that every piece of claimed wall is built with an arrow slit which lets creatures inside the dungeon see if there are enemies just behind the wall.

Well, the way I see it, it is even more logical. Claimed tiles give vision, be an allied creature on it or not. That's the keeper magic. And to say it is the same for claimed walls doesn't look weird to me.

> and would find not knowing what exactly is coming for me be both more logical and more ominous.

Well, usually, you only see the workers putting pression trying to rush to you. But you don't know the army that will come if they can go through your wall...
